src/main/java/frc/robot/subsystems/led/LED.java

```diff
@@ -94,6 +98,12 @@ public void setRandomPattern() {
     setControllerPower(getRandomPattern());
   }
 
+  public void startEmergencyLED(){
+    DriverStationLogger.logToDS("Starting emergency LED");
```

Comment:
DriverStationLogger.logToDS("Starting emergency LED");
I think you should remove this and in the DriverStationLogger add in the logErrorToDS method:
Robot.led.startEmergencyLED();

AND add stopEmergencyLED method:
`public void stopEmergencyLED() {
    DriverStationLogger.logToDS("LED emergency disabled");
    setColor(LEDColor.Off);
    isInEmergency = false;
}`